add RealEstate Scrap

KPilosyan opened the pull request.

KPilosyan committed:
add RealEstate Scrap

hpilosyan commented on server/logs/log-11-11-2021.log:
```diff
@@ -0,0 +1,50 @@
+11-11-2021 16:30:05 [error]: ScraperController:start:FAILED {"ERROR":""}
```
Do not commit your logs

hpilosyan commented on server/scraper/ListAmScraper.js:
```diff
@@ -1,183 +1,151 @@
 import cheerio from 'cheerio';
```
Rename this file to real estate scraper

hpilosyan commented on server/services/VehicleEstimateService.js:
```diff
@@ -45,3 +45,17 @@ export default class VehicleEstimateService {
         }
     }
 }
+// const buildings = {
```
What's this?

KPilosyan replied:
Nothing. Forgot it was there.

KPilosyan committed:
Final Version

Main change is that the code now awaits for all urls of a page, then sends requests all together. It's much faster.
